scripts/compact-junit.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from junitparser import JUnitXml, Failure
+
+
+def main():
+    junit_xml_file = sys.argv[1]
+    suite = JUnitXml.fromfile(junit_xml_file)
+    for case in suite:
+        if not isinstance(case.result, Failure):
+            case.system_out = ''
+            case.system_err = ''
+    suite.write()
```

Comment:
I'm curious: This script is simply reading and writing back the data. How it is "compressing" then? maybe a short comment explaining this would be useful.